templates/base_layout.html

```diff
@@ -40,9 +40,11 @@
   <link rel="preload" as="font" type="font/woff2" href="https://assets.ubuntu.com/v1/f8097dea-Ubuntu-LI_W.woff2" crossorigin>
   <link rel="preload" as="font" type="font/woff2" href="https://assets.ubuntu.com/v1/fff37993-Ubuntu-R_W.woff2" crossorigin>
 
-  <meta name="description" content="{% block meta_description %}{% endblock %}">
+  
+  <meta name="description" content="{% block meta_description %}Open source Python operators for Kubeflow on-rails. For workstations, multi-cloud and edge.{% endblock %}">
```

Comment:
Can you add this description to the index page please.

Reply:
Added it 👍